docs/docs/fallback-handoff.mdx

```diff
@@ -231,80 +215,147 @@ class ActionDefaultFallback(Action):
 ```
 
 
-
 <img alt="Intent Mappings" src={useBaseUrl("/img/intent_mappings.png")} width="240" />
 
-## Out of scope intent
 
-It is good practice to also handle questions you know your users may ask, but for which you haven't necessarily implemented a user goal yet.
 
-You can define an `out_of_scope` intent to handle generic out of scope requests, like “I'm hungry” and have
-the bot respond with a default message like “Sorry, I can't handle that request”:
+### Two-Stage Fallback
 
-```md
-* out_of_scope
-  utter_out_of_scope
-```
+In some cases, you will want to attempt to disambiguate the user's message by asking
```

Comment:
not sure i would say "in some cases" - i would more say that this is the more user friendly option and should be used. the basic fallback is more just a "quick" solution, if you want your users to have the best experience, then this two stage fallback should be used.